src/redux/slices/settingsSlice/types.ts

```diff
@@ -87,3 +87,9 @@ export interface SettingsSlice {
   bitcoin: Bitcoin | null
   usedBitcoinAddresses: { [key: string]: string }
 }
+
+export class TraceError extends Error {
```

Comment:
I guess this is not necessary, we have `GlobalErrorContext` if you wish to block developer from using the app unless the `TRACE_ID` is set.

Reply:
I removed it.